website/content/blog/2024-09-11-meeting.md

```diff
@@ -0,0 +1,134 @@
+:::
+
+### Release Improvements
+
+Like the Content Pipeline improvements above, the same view has also been turned to the way we release MonoGame today via NuGets.  We recognize this might not be a 100% fit for all as NuGet does impose some limitations, especially for offline use.
+
+To this end, we have also kicked off two other streams of work:
+
+- [Update build pipeline to generate a "Complete" Source package for releases](https://github.com/MonoGame/MonoGame/issues/8485)
+
+    Currently, the Source package that GitHub automatically generates ONLY contains the MonoGame source, it does not include submodules or dependencies.
+    This change will generate a zip that includes **ALL** source, including submodules and dependencies so that it can be dropped in a project and referenced.
+    Source access will allow a project to be completely self-contained and build the framework on demand.
+
+
+- [Update build pipeline to generate a "Binaries" Source package for releases](https://github.com/MonoGame/MonoGame/issues/8486)
+
+    Currently the Binaries package that GitHub automatically generates ONLY contains the MonoGame binaries, it does not include submodules or dependencies.
+    This change will generate a zip that includes **ALL** binaries and dependencies so that it can be dropped in a project and referenced.
+    Perfect for automation or offline scenarios where snapshot builds of dependencies are needed without modification. (instead of NuGet)
+
+Hopefully, this will open up new avenues for advanced projects and pipelines.
+
+::: important
+If you have a suggestion on how to improve this or add more flexibility at runtime, **NOW** is the time to have your voice heard by commenting or making suggestions in the issues above.
+:::
+
+### Bounties, Bounties, Bounties
+
+Work continues on all existing bounties, but these updates were at the top of mind for discussion.
+
+- DX12 / GDK support is a go
+    The work for the DX12/GDK update is now complete, required dependencies for the "platform" are being merged into the public MonoGame this week. Once in, the private Xbox repo will be published with the new updates, then finally the public MonoGame / GDK updates can be published.  Expect this amazing release imminently (more imminently than previously imminent, meaning very soon (tm))
+
+- The final gap in updating the Content Pipeline dependencies is complete
+    The [Basis Universal](https://github.com/MonoGame/MonoGame/pull/8456) bounty was the final milestone in generating true cross-platform capabilities needed for an updated Content Pipeline.  All that remains now is the work to stick it all together.  But even that will not be the end, as once the NativeAoT work is complete, it will open up even more avenues for enhancement.
+
+- NativeAoT platform is 99% done!

```

Comment:
Regarding the NativeAot being 99% done.. Does that need clarification that is does not mean anyone can even use it yet. Some people will only read the titles after all.